web/turkey/survey/static/survey/js/auditors/clicks_specific.js

```diff
@@ -0,0 +1,33 @@
+var AuditorClicksSpecifics = {
+    clicks_specifics: [],
+    log_click_content: function (e) {
+        var dom = {
+            'type'  :   e.target.nodeName.toLowerCase(),
+            'id'    :   e.target.id != '' && e.target.id != undefined 
```

Comment:
When you make corresponding Python models, make sure to see the appropriate fields nullable to avoid problems.